### PR TITLE
Macaw UI update

### LINKED ISSUE #2643: Update Macaw UI

### PR DESCRIPTION
**Pull request by Cloud11PL**
I want to merge this change because it fixes #2643

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://apps.saleor.io
SALEOR_APPS_ENDPOINT=https://apps.saleor.io/api/saleor-apps

### Do you want to run more stable tests?
To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [ ] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants

CONTAINERS=1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "@material-ui/lab": "^4.0.0-alpha.58",
     "@material-ui/styles": "^4.11.4",
     "@reach/auto-id": "^0.16.0",
-    "@saleor/macaw-ui": "0.6.5",
+    "@saleor/macaw-ui": "0.6.7",
```
Comment (andrzejewsky) [inline package.json:33]: keep it linked on this branch

File: package.json [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
     "@glideapps/glide-data-grid": "^5.0.0",
-    "@material-ui/core": "^4.11.4",
+    "@material-ui/core": "^4.12.4",
     "@material-ui/icons": "^4.11.2",
-    "@material-ui/lab": "^4.0.0-alpha.58",
+    "@material-ui/lab": "^4.0.0-alpha.61",
     "@material-ui/styles": "^4.11.4",
     "@reach/auto-id": "^0.16.0",
-    "@saleor/macaw-ui": "0.6.5",
+    "@saleor/macaw-ui": "git@github.com:saleor/macaw-ui.git#3337aa0",
```
Comment (NyanKiyoshi) [inline package.json:33]: Use full hash, short hash is dangerous

**Review by NyanKiyoshi — CHANGES_REQUESTED** [review]

**Comment by szczecha** [comment]
- [ ] Should an eye icon have a frame? image

- [ ] Shouldn't back be a secondary button? 
image

- [ ] a lot of empty space between the filters 
image

- [ ] The search input should be the same height as the buttons 
image

- [ ] Edit staff user, dark mode, default avatar does not have initials
image

- [ ] Different alignment of metadata inputs (e.g. views: clients, products)
image

**Comment by szczecha** [comment]
- [ ] Inconsistency in checkboxes colors
image
Controls form 0.6.7 macawUI
image

**Comment by karolm-saleor** [comment]
Another thing that comes to mind here is ponglish.
If we choose one particular language all the suggestions and names should match the same language

**Comment by karolm-saleor** [comment]
- [ ] When I enter Catalog - Products and select the first one on black mode eveytime the view is visible for 2 second then after a moment it crashes
Screenshot 2022-11-22 at 15 01 54

**Review by krzysztofzuraw — APPROVED** [review]

**Review by NyanKiyoshi — DISMISSED** [review]

**Review by NyanKiyoshi — APPROVED** [review]
Only reviewed non .ts(x) files

**Pull request merged**